docs/docs/build/getting-started/helloworld.mdx

```diff
@@ -5,23 +5,30 @@ sidebar_position: 2
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-# Hello World
+# Hello Bitcoin
 
 This guide provides step-by-step instructions on how to deploy smart contracts on BOB.
 
 We will provide instructions for using Remix and Foundry.
 
+- [Developer Starters Kit](https://github.com/bob-collective/DevStarterKit) the kit serves as the foundation to develop on BOB.
```

Comment:
I think the URL needs updating - github has a redirect but best to have it at the correct URL.